lib/danger/request_sources/github.rb

```diff
@@ -174,12 +192,12 @@ def parse_comment(comment)
         violations[kind] = violations_from_table(table)
       end
 
-      violations
+      violations.reject { |k, v| v.empty? }
```

Comment:
Can we leave `k` here? Or if not, rename to `_` to indicate it's not being used?